helios-client/src/test/java/com/spotify/helios/common/JobValidatorTest.java

```diff
@@ -268,6 +280,25 @@ public void testInvalidImagesFail() throws Exception {
                  validator.validate(b.setImage(foos + "/bar").build()));
   }
 
+   @Test
+   public void testInValidHostnamesFail() {
+     final Job.Builder b = Job.newBuilder().setName("foo").setVersion("1").setImage("bar");
+
+     // 64 chars
+     final String toolonghostname = Strings.repeat("hostname", 8);
```

Comment:
The checkstyle plugin doesn't check test code, but we'd like to keep these lines here to a maximum of 100 chars.